wire-ios-data-model/Source/ManagedObjectContext/NSManagedObjectContext+SyncResources.swift

```diff
@@ -31,7 +31,7 @@ public extension NSManagedObjectContext {
     }
 
     /// use to trigger sync resources after some CoreData migrations
-    func setMigrationNeedsSyncResources() throws {
+    func setMigrationNeedsSyncResources() throws { // TODO: needed on import?
```

Comment:
as we'll always perform slow sync after importing a backup, I think this can be left out.